index.md

```diff
@@ -0,0 +1,48 @@
+# Data manipulation, analysis and visualisation in Python
+
+## Introduction
+
+The handling of data is a recurrent task for most scientists. Reading in experimental data, checking its properties and creating visualisations may become tedious tasks. Hence, increasing the efficiency in this process is beneficial for many scientists. Spreadsheet based software lacks the ability to properly support this process, due to the lack of automation and repeatability. The usage of a high-level scripting language such as Python is ideal for these tasks. 
+
+This course trains students to use Python effectively to do these tasks. The course focuses on data manipulation and cleaning, explorative analysis and visualisation using some important packages such as Pandas, Numpy and Matplotlib.
+
+The course has been developed as a specialist course for the Doctoral schools of Ghent University, but can be taught to others upon request. 
+
+## Course info
+
+### Aim & scope
+
+This course is intended for researchers that have at least basic programming skills in Python. It targets researchers that want to enhance their general data manipulation and analysis skills in Python. 
+
+The course does not aim to provide a course in statistics or machine learning. It aims to provide researchers the means to effectively tackle commonly encountered data handling tasks in order to increase the overall efficiency of the research. 
+
+### Program
+
+The course is organized as a three day course with following program:
+
+- **Day 1** - Basic concepts and introduction to Pandas: Some essential concepts of the Python computing language are rehearsed. Setting up the programming environment with the required packages using the conda package manager and an introduction of the Jupyter notebook environment are covered. The day closes with an introduction to the data analysis package Pandas.
+- **Day 2** - Advanced Pandas: The usage of Pandas for different data cleaning and manipulation tasks is taught and the acquired skills will immediately be brought into practice to handle real-world data sets. Besides, the automation of these tasks as a reusable function or module will be tackled. Applications include time series handling, categorical data, merging data, ...
+- **Day 3** - Combining Pandas and Matplotlib: The process of data cleaning is combined with the creation of plots for data exploration.  Essential functionalities of Matplotlib are taught and directly applied. 
```

Comment:
The content for (division between) the different days is not really representative anymore .. :), but yeah, that's another issue.

Reply:
Adapting it to make it more repesentative or removing this stion are both fine for me. Although I think it is good to show it is a three day course.